src/ts/utils/__tests__/getCollectionSortKeys.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  defaultSortKey,
+  collectionSortKeys,
+  getValidSortkey
+} from '../getCollectionSortKeys'
+
+describe('getCollectionSortKeys returns valid sort key configuration', () => {
+  test('should have a default sort key', () => {
```

Comment:
This test's `describe` block mentions `getCollectionSortKeys`, but the test has nothing to do with that method. Break out each method exported from the util into its own `describe` block to better organize the tests by function